libs/filamat/src/GLSLPostProcessor.cpp

```diff
@@ -38,6 +40,42 @@ using namespace spvtools;
 
 namespace filamat {
 
+namespace {
+
+    std::string addMaybeUnusedToVariables(const std::string& mslShader) {
+        constexpr std::string_view unusedVariables[] = {
+            "float3x3 shading_tangentToWorld;",
+            "float3 shading_position;",
+            "float3 shading_view;",
+            "float3 shading_normal;",
+            "float3 shading_geometricNormal;",
+            "float3 shading_reflected;",
+            "float shading_NoV;",
+            "float2 shading_normalizedViewportCoord;",
+            "float3 shading_clearCoatNormal;",
+            "float filament_lodBias = frameUniforms.lodBias;",
+            "float level0 = fast::clamp(floor(log2(ssSliceRadius)) - 3.0, 0.0, setup.maxLevel);",
```

Comment:
This originates from an actual unused local variable in `ssct.fs`. Let's remove that instead.

Reply:
https://github.com/shapr3d/filament/pull/100/commits/f9aad2032cea8868772f17faeaa23098d0d823fd